src/connection/BaseConnectionOptions.ts

```diff
@@ -133,6 +133,11 @@ export interface BaseConnectionOptions {
          */
         readonly duration?: number;
 
+        /**
+         * Configurable Table Name or query-result-cache as Default.
+         */
+        readonly cacheTableName?: string;
```

Comment:
We should move this option inside `options` property as this is setting only for "database" type of cache. 

Also `tableName` should be sufficient. It is already inside of `cache` object so `cache` prefix is not necessary.